.gitignore

```diff
@@ -7,8 +7,9 @@ static-collected
 
 .DS_Store
 static/.DS_Store
-
 hc-venv/
-hc/.local_settings
 
 .vscode
+.idea/
+venv/
```

Comment:
Remove this line. You virtual env must be outside the project folder. I am assuming you followed the  README to set up the project.

Reply:
I have uptaded this accordingly

Reply:
I have edited that.